metsrw/exceptions.py

```diff
@@ -1,3 +1,4 @@
+# -*- coding: utf-8 -*-
```

Comment:
# -*- coding: utf-8 -*-

Not all files have been updated, I think `__init__` and the stuff under `plugins/` is missing.